docs/UserGuide.md

```diff
@@ -676,6 +724,8 @@ Deletes a meeting from the calendar when a meeting is cancelled
 - Command: `delete-m`
 - Format: `delete-m <INDEX_NUMBER>`
 
+> - Deletes the meeting at the specified `INDEX_NUMBER`, which refers to the index shown on the displayed meeting list. The index must be a positive integer 1, 2, 3...
```

Comment:
Looks slightly off with the > followed by a bullet, maybe can remove the bullet/dash.

Reply:
oh this was standardizing for all edit and delete command wanted to separate the bullets form command and format